pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -192,6 +193,10 @@ private static void validateValidationConfig(TableConfig tableConfig, @Nullable
       }
     }
 
+    if (validationConfig.isNullTimeColumnHandlingEnabled()) {
```

Comment:
There can be multiple date time fields. Do we want to handle the null time value for only the time column or all the date time fields?

Reply:
Only the time column.